README.md

```diff
@@ -1,2 +1,46 @@
-# First-Capstone-Project
-This is going to be my first capstone project built at Microverse.
+# Crimson Dawn Car Meeting | Capstone Project - Module 1
+
+> This is the final version of my project for the 5th week, Module 1. In this project I used all the knowledge I gathered from past weeks, and blended them nicely. [Link to video documentation](https://www.loom.com/share/67fb8fc898d24f04a047aa5a87d0e247) 
+![screenshot](assets/images/demodesk.png)
+
+My challenge was to build a conference website from scratch, using the template from Behance. [Live Demo](https://adriancsm.github.io/First-Capstone-Project/). 😊
+
+### Features implemented:
+* Dynamically loading the speakers list
+* Used Bootstrap
+* Mobile menu
+* Responsive design
+* Modern look
+
+## Built With
+- HTML5
+- CSS3 (Flex and Media queries)
+- Bootstrap styling (A popular CSS Framework for developing responsive and mobile-first websites.)
+- Lighthouse (An open-source, automated tool for improving the quality of web pages. It has audits for performance, accessibility, progressive web apps, SEO and more).
+- Webhint (A customizable linting tool that helps you improve your site's accessibility, speed, cross-browser compatibility, and more by checking your code for best practices and common errors).
+- Stylelint (A mighty, modern linter that helps you avoid errors and enforce conventions in your styles).
+- ESlint (A mighty, modern linter that helps you avoid errors and enforce conventions in JavaScript codes)
+
+To get a local copy up and running follow these simple example steps.
```

Comment:
- [ ] Kindly add some steps on how the user can use your project locally so your README is documented professionally.